src/types/models/users.ts

```diff
@@ -74,12 +74,17 @@ export type UserPreferences = {
   };
   blockedUsers: UserId[];
   onboarding: {
-    dreamJobs: OnboardingMetaDocumentData['dreamJobs'];
-    dreamTeachers: OnboardingMetaDocumentData['dreamTeachers'];
-    skills: OnboardingMetaDocumentData['skills'];
+    question1: OnboardingAnswer;
+    question2: OnboardingAnswer;
+    question3: OnboardingAnswer;
```

Comment:
if you make these names the same name as the meta doc you can directly map them. 

else you'll have to create a new map on the frontend

just make this
onboarding: {
   onboardingQuestionOne: OnboardingAnswer